query.go

```diff
@@ -772,6 +773,8 @@ func scanRow(cols []string, rows *sql.Rows) (Result, error) {
 		return nil, fmt.Errorf("Error scanning row: %s", err)
 	}
 
+	columnTypes, _ := rows.ColumnTypes()
```

Comment:
we should check for errors and return with the error like in line 773

Reply:
FIXED